plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/di/MultiBindingDefaults.java

```diff
@@ -88,7 +97,22 @@ private MultiBindingDefaults() {}
       TriggerEdgeCreationAction.class,
       UpdateModelAction.class);
 
+   public static final List<Class<? extends Action>> DEFAULT_CLIENT_ACTIONS = Lists.newArrayList(
+      SetClipboardDataAction.class,
```

Comment:
It looks like this list is not complete. There are a couple of other client actions (ike. Center, FitToScreen, Select, Expand, ServerStatusAction etc.)

We should also double check the ActionHandlers e.g. the `SetEditModeActionHandler` implicitly registers the `SetEditModeAction` which has to be registered as client action as well.

Reply:
Very good point! As discussed, I registered the necessary actions as client actions and also removed up some unused actions, cf also https://github.com/eclipse-glsp/glsp-client/pull/74.

I also added a logging example for the server-side action dispatching, cf. https://github.com/eclipse-glsp/glsp-examples/pull/60

Reply:
Very good observation! I tested this and you are right, the DIActionRegistry already captures all handled actions through the action and operation handler bindings. As a result, the default action binding was removed altogether. I pushed a new commit to reflect that. Thank you!